kolla/defaults.yml

```diff
@@ -345,6 +346,10 @@ enable_k3s: no
 enable_zun: no
 enable_zun_compute_k8s: "{{ enable_zun | bool and enable_k3s | bool }}"
 blazar_enable_device_plugin_k8s: "{{ enable_k3s | bool }}"
+# K8S worker tainting
+zun_enforce_worker_taint: false
```

Comment:
should this be `zun_tolerate_worker_taint`? afaik it will not prevent scheduling even if no workers are tainted?

Reply:
agreed